maestro-xcuitest-driver/src/main/kotlin/util/CommandLineUtils.kt

```diff
@@ -15,6 +15,7 @@ object CommandLineUtils {
         ) "NUL" else "/dev/null"
     )
 
+    @Deprecated("use runCommand(listOf(...)) instead")
```

Comment:
Lets maybe add why it's deprecated. A small description, if not here then on internal slack